lineman/app/components/thread_page/proposal_collapsed/proposal_collapsed.haml

```diff
@@ -1,9 +1,11 @@
 %a.proposal-collapsed.media{href: '', id: 'proposal-{{proposal.key}}'}
   .sr-only
     {{proposal.name}}.
-    %closing_in{time: 'proposal.closingAt'}.
+    %closing_in{ng-if: 'proposal.isActive()', time: 'proposal.closingAt' }.
```

Comment:
Seeing this pattern 3 times makes me wonder if the closing_in directive shouldn't just handle this itself (it's already handling whether the time is before or after right now..)